html_table_parser/parser.py

```diff
@@ -36,7 +36,7 @@ def __init__(
         self.named_tables = {}
         self.name = ""
 
-    def handle_starttag(self, tag, attrs):
+    def handle_starttag(self, tag: str, attrs: str) -> None:
```

Comment:
attrs should be a list, not a str here.

Reply:
Thank you for the request. One can ask why I didn't use `list` and imported the `List` from `typing`. The reason is that "deep type hinting" cannot be done with the built-in `list` in `python3.5`.